Dockerfile

```diff
@@ -1,17 +1,12 @@
 FROM ubuntu:16.04
-MAINTAINER Event Store LLP <ops@geteventstore.com>
 
-ENV ES_VERSION=5.0.0-1 \
-    DEBIAN_FRONTEND=noninteractive \
+ENV ES_VERSION=5.0.0-1 \    
     EVENTSTORE_CLUSTER_GOSSIP_PORT=2112
 
 RUN apt-get update \
     && apt-get install tzdata curl iproute2 -y \
-    && curl -s https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh | bash \
-    && apt-get install eventstore-oss=$ES_VERSION -y \
-    && apt-get autoremove \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    && curl -fSL -s https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh | bash \
```

Comment:
Looking at the contents of https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh, I'm not comfortable with this script being used even if it includes more verification -- as far as I can tell, the script amounts to essentially `curl ... | apt-key add - && curl https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/config_file.list?os=ubuntu&dist=bionic&source=script > /etc/apt/sources.list.d/EventStore_EventStore-OSS.list`, and it doesn't do any real verification of anything downloaded.  What I'd rather see instead is the PGP key being fetched in such a way that the full fingerprint is verified (from a PGP keyserver with `--recv-keys ...`, for example), and either embedding https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/config_file.list?os=ubuntu&dist=bionic&source=script or just embedding `deb https://packagecloud.io/EventStore/EventStore-OSS/ubuntu/ bionic main` directly (since `deb-src` isn't necessary and the path doesn't seem likely to change).